app/views/calculator/payment_today_form.scala.html

```diff
@@ -68,15 +70,15 @@
             name = "amount",
             value = dataForm.data.get("amount"),
             hint = Some(Hint(
-                content = Text(messages("ssttp.calculator.form.payment_today.hint", currency(debits.map(_.amount).sum)))
+                content = Text(messages("ssttp.calculator.form.payment_today.hint", currency(debits.map(_.amount).sum - 2)))
             )),
             label = Label(
                 forAttr = Some("amount"),
                 content = Text(messages("ssttp.calculator.form.payment_today.title")),
                 classes = "govuk-!-font-weight-bold govuk-label--xl govuk-!-margin-bottom-9",
                 isPageHeading = true
             ),
-            errorMessage = dataForm.error("amount").map(error => errorMessageWithDefaultStringsTranslated(content = Text(messages(error.message)))),
+            errorMessage = dataForm.error("amount").map(error => errorMessageWithDefaultStringsTranslated(content = Text(messages(error.message, currency(debits.map(_.amount).sum - 2))))),
```

Comment:
couldn't you just do `messages(error.message, error.args: _*)` to get the parameters in the message? That would make it work for all error messages if more than one has parameters